SU-SWS/stanford_profile
# Removed unnecessary field permission third party settings

# READY FOR REVIEW

# Summary
- Removed field permission third party settings
- see https://github.com/SU-SWS/stanford_profile_helper/pull/26

# Need Review By (Date)
- :man_shrugging: 

# Urgency
- low

# Steps to Test
1. checkout this branch.
1. do an install or config import
1. see that it still works like normal.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
